### DISCUSSION TITLE
fps match camera fps?

### DISCUSSION THREAD

**Discussion by vajonam** [blakeblackshear/frigate#6158] [Ask A Question] [answered]
does the detect fps need to match the camera's naive fps? what the side effects of these two not in sync? does ffmpeg change the frame rate to match what is defined `detect` config?

**Comment by NickM-27** [comment, marked as answer]
No, they do not need to match. ffmpeg will limit the fps that is sent to motion / object detection to whatever is set in `detect` configuration

the downside is that you still have to decode the extra frames and then they just get discarded, so there is some wasteful resource usage
